tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py

```diff
@@ -68,6 +68,8 @@ def test_import_edit_export_contact(self):
         self.assertEqual(self.sim_contact['tel'][0]['value'], self.contact.tel['value'])
 
     def tearDown(self):
-        self.data_layer.delete_sim_contact(self.sim_contact['id'])
+        for contact in self.data_layer.sim_contacts:
+            if contact['name'][0].startswith('Name'):
+                self.data_layer.delete_sim_contact(contact['id'])
```

Comment:
:+1: Let's [prefix the names there](https://github.com/mozilla-b2g/gaia/pull/31474/files#diff-087812ad7aa03d9cdb90e5349e7bcd6aR14).